src/fuzz/fuzz-env-file.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+max_len = 65535
```

Comment:
This isn't going to work when AFL is used as a fuzzing engine so ClusterFuzz will keep opening issues you're trying to avoid. They recommend checking whether the length passed to `LLVMFuzzerTestOneInput` makes sense in fuzz targets.

Reply:
https://github.com/google/oss-fuzz/issues/1846#issuecomment-425443711